### ISSUE TITLE
Solution to LINES expects newlines but doesn't say so

### ISSUE THREAD

**Issue by bbugh** [workshopper/stream-adventure#96]
I'm not a newbie, and this was confusing for quite a while. I knew my solution was correct, and could not for the life of me figure out why it wasn't working. The only flaw was the missing `+ "\n"`. It seems worthwhile to explicitly mention this in the instructions as a hint. 

I think it's more confusing because the example code doesn't show a newline, so I kept looking at that wondering why it wasn't working.

``` javascript
    var split = require('split');
    process.stdin
        .pipe(split())
        .pipe(through(function (line) {
            console.dir(line.toString());  // probably should add a + "\n" here to be helpful?
        }))
    ;
```

In hindsight, it should be obvious (because I've done streams a thousand times), but remembering those explicit details while learning new concepts from an "incorrect" example can be challenging!

Also this is so awesome.

**Comment by Rybadour** [comment]
Agreed, when verifying your solution the naive implementation just results in null for all the actual values.

I assume it's because the buffer flushes with a new line so even though you queue it the right string (minus the newline) it still doesn't get any data.

**Comment by anupamkumar** [comment]
This stumped me too for a while. But then again, we all had made the assumption that newline character will automatically be added upon joining! LOL.

**Comment by kalinchernev** [comment]
PR open to specify the possible scenarios, feedback is welcome there.

**Comment by ccarruitero** [comment]
fixed in #193
